sdk/confidentialledger/azure-security-confidentialledger/src/test/java/com/azure/security/confidentialledger/GetConsortiumMembersTests.java

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.confidentialledger;
+
+import com.azure.core.credential.AccessToken;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.netty.NettyAsyncHttpClientBuilder;
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.http.rest.Response;
+import com.azure.core.test.TestMode;
+import com.azure.core.util.BinaryData;
+import com.azure.core.util.Configuration;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslContextBuilder;
+import reactor.core.publisher.Mono;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.time.OffsetDateTime;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public final class GetConsortiumMembersTests extends ConfidentialLedgerClientTestBase {
```

Comment:
There looks like a lot of duplicated code in your ConfidentialLedgerClientTestBase and here wrt to initialization.
It is easier to reason about if you have all the tests pertaining to ConfidentialLedgerClient in a single file, `ConfidentialLedgerClientTests.java` and then have separate methods for each test case for that.

```java
public class ConfidentialLedgerClientTests extends ConfidentialLedgerClientTestBase {
   @Test
   public void getConsortiumMembersTest() {
   }

   @Test
   public void getEnclaveQuotesTest() {
   }
}
```

Reply:
Right! I agree! Unfortunately, I can't get it to work when I add the code to the TestBase. I am hoping to refactor it later :)

Reply:
Ahh, I see!  It would be good to create a GitHub issue to address this so it doesn't get lost.

Reply:
I will create a sprint board issue for myself. Where should I create the issue?

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/new